packages/suite/src/hooks/settings/useNetworkSupport.ts

```diff
@@ -5,14 +5,28 @@ import { hasBitcoinOnlyFirmware } from '@trezor/device-utils';
 import { arrayPartition } from '@trezor/utils';
 
 import { useSelector } from 'src/hooks/suite';
-import { selectIsDebugModeActive } from 'src/reducers/suite/suiteReducer';
+import {
+    selectIsDebugModeActive,
+    selectHasExperimentalFeature,
+} from 'src/reducers/suite/suiteReducer';
+import { EXPERIMENTAL_L2_NETWORKS } from 'src/actions/suite/constants/suiteConstants';
+import { selectEnabledNetworks } from 'src/reducers/wallet/settingsReducer';
 
 export const useNetworkSupport = () => {
     const device = useSelector(selectSelectedDevice);
     const isDebug = useSelector(selectIsDebugModeActive);
     const deviceSupportedNetworkSymbols = useSelector(selectDeviceSupportedNetworks);
+    const isEthereumL2SupportEnabled = useSelector(
+        selectHasExperimentalFeature('ethereum-l2-support'),
+    );
+    const enabledNetworks = useSelector(selectEnabledNetworks);
 
-    const mainnets = getMainnets(isDebug);
+    const mainnets = getMainnets(isDebug).filter(
+        network =>
+            isEthereumL2SupportEnabled ||
+            !EXPERIMENTAL_L2_NETWORKS.includes(network.symbol) ||
+            enabledNetworks.includes(network.symbol),
```

Comment:
Is this correct? In my point of view in mainnets should be all symbols (except L2 if not turned on) regardless enabledNetworks. enabledNetworks are defined by user actions, if I get it right. E.g. VTC is not in enabledNetworks, but it is supported network.

Reply:
you are right, last minute change

Reply:
9f439c53605ebe7d6b78d755acc509dc37c44d61